### PR TITLE
vSphere support update

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

**Comment by vikram-redhat** [comment]
Ok to merge.

File: modules/installation-vsphere-infrastructure.adoc [hunk 1]
```diff
@@ -5,5 +5,22 @@
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 6.5 or
-6.7 instance.
\ No newline at end of file
+You must install the {product-title} cluster on a VMware vSphere version 6.5U2
+or 6.7U2 or later instance.
+
+VMware strongly recommends using vSphere Version 6.7U2 or later with your
```
Comment (katherinedube) [inline modules/installation-vsphere-infrastructure.adoc]: I'd suggest removing "strongly"

File: modules/installation-vsphere-infrastructure.adoc [hunk 2]
```diff
@@ -5,5 +5,22 @@
...
+{product-title} cluster.
+vSphere 6.7U2 includes:
+
+* Support for VMware NSX-T v2.5 upon its release
```
Comment (katherinedube) [inline modules/installation-vsphere-infrastructure.adoc]: I'd drop the version reference for NSX-T. Would be easier to maintain.

**Comment by kalexand-rh** [comment]
@katherinedube, thanks for the suggestions! I've incorporated them.

**Comment by kalexand-rh** [comment]
I haven't heard more feedback, so I'm going to merge. Please tag me in follow-up PRs, if necessary.

**Pull request merged**